etc/webapi.xml

```diff
@@ -205,4 +205,11 @@
             <resource ref="Magento_Cart::manage" />
         </resources>
     </route>
+    <!--Get product image url by product id, imageId can be as parameter -->
+    <route url="/V1/bolt/boltpay/product/:productId/image-url" method="GET">
```

Comment:
Maybe imageUrl ? What's the proper naming from magento?

Reply:
magento uses camel case for variables, and "dash" for path, core examples:
/V1/products/attribute-sets/sets/list
/V1/products/:sku/group-prices/:customerGroupId/tiers
/V1/products-render-info